resources/js/components/people/ContactSearch.vue

```diff
@@ -49,7 +49,19 @@ export default {
       if (contact.item.id > 0) {
         window.location = contact.item.route;
       } else {
-        window.location = 'people/add';
+        const names = contact.item.keyword.split(' ').map(name =>
+          name.charAt(0).toUpperCase() + name.substr(1).toLowerCase()
+        );
+        const first_name = names[0];
```

Comment:
Everybody don't use "first_name last_name", actually there is a config for that in the settings page.

Reply:
In the last commit this is resolved